### ISSUE TITLE
Question about in determining train_steps/val_steps in helper_tool.py

### ISSUE THREAD

**Issue by amiltonwong** [QingyongHu/RandLA-Net#7]
Hi, @QingyongHu ,

Thanks a lot for the release of you package. I had tried the training on S3DIS dataset and it runs smoothly.

One place confusing me is the train_steps/val_steps in [helper_tool.py](https://github.com/QingyongHu/RandLA-Net/blob/master/helper_tool.py#L51-L54).  Why isn't the train_steps/val_steps configured as the total size of the training/val set divided by the batch_size?

It is shown that the train_steps/val_steps is configured as the same value for all 6 area sets in S3DIS. Will it miss some parts for longer set such as Area-5?

THX!

**Comment by QingyongHu** [comment]
Hi, @amiltonwong. Thanks for your interest. This is just for fast training and validation. We based on the assumption that if the trained model can get a good result on the part of the validation set, this model is likely to achieve a good result on the whole validation set. 

You are welcome to evaluate the whole train/validation set (refer to the implementation on main_SemanticKITTI),  but it would be a little bit slow.

**Comment by amiltonwong** [comment]
Thanks @QingyongHu for your reply.

**Comment by a18700** [comment]
Hi there, i want to clarity that if the experimental results of S3DIS and Semantic3D are based on fast-implementation. Is the whole data is required for reproducing results in the paper?
